packages/devtools_app/lib/src/screens/logging/logging_screen_v2/logging_screen_v2.dart

```diff
@@ -104,104 +85,71 @@ class _LoggingScreenState extends State<LoggingScreenBody>
   Widget build(BuildContext context) {
     return Column(
       children: [
-        _buildLoggingControls(),
-        const SizedBox(height: intermediateSpacing),
+        Row(
+          children: [
+            Expanded(
+              child: TextField(
```

Comment:
use DevToolsClearableTextField for both of these. Fine to implement functionality later

Reply:
I'll implement this in the next one.